pkg/cmd/roachtest/test_registry.go

```diff
@@ -63,29 +65,30 @@ type OwnerMetadata struct {
 // roachtestOwners maps an owner in code (as specified on a roachtest spec) to
 // metadata used for github issue posting/slack rooms, etc.
 var roachtestOwners = map[Owner]OwnerMetadata{
-	OwnerSQLExperience: {SlackRoom: `sql-experience`, ContactEmail: `rafi@cockroachlabs.com`,
+	OwnerSQLExperience: {SlackRoom: `sql-experience`, Mention: []string{`@cockroachdb/sql-experience`},
 		TriageColumnID: 7259065,
 	},
-	OwnerBulkIO: {SlackRoom: `bulk-io`, ContactEmail: `david@cockroachlabs.com`,
+	OwnerBulkIO: {SlackRoom: `bulk-io`, Mention: []string{`@cockroachdb/bulk-io`},
 		TriageColumnID: 3097123,
 	},
-	OwnerCDC: {SlackRoom: `cdc`, ContactEmail: `ajwerner@cockroachlabs.com`,
+	OwnerCDC: {SlackRoom: `cdc`, Mention: []string{`@cockroachdb/cdc`},
 		TriageColumnID: 3570120,
 	},
-	OwnerKV: {SlackRoom: `kv`, ContactEmail: `andrei@cockroachlabs.com`,
+	OwnerKV: {SlackRoom: `kv`, Mention: []string{`@cockroachdb/kv`},
 		TriageColumnID: 3550674,
 	},
-	OwnerPartitioning: {SlackRoom: `partitioning`, ContactEmail: `andrei@cockroachlabs.com`,
-		// Partitioning issues get sent to the KV triage column for now.
+	// This is an alias for the KV team.
+	OwnerPartitioning: {SlackRoom: `kv`, Mention: []string{`@cockroachdb/kv`},
 		TriageColumnID: 3550674,
 	},
-	OwnerSQLExec: {SlackRoom: `sql-execution-team`, ContactEmail: `alfonso@cockroachlabs.com`,
+	// TODO(jordanlewis): that team doesn't exist any more, fix things up.
+	OwnerSQLExec: {SlackRoom: `sql-execution-team`, Mention: []string{`@cockroachdb/sql-queries`},
 		TriageColumnID: 6837155,
 	},
-	OwnerSQLSchema: {SlackRoom: `sql-schema`, ContactEmail: `lucy@cockroachlabs.com`,
+	OwnerSQLSchema: {SlackRoom: `sql-schema`, Mention: []string{`@cockroachdb/sql-schema`},
 		TriageColumnID: 8946818,
 	},
-	OwnerStorage: {SlackRoom: `storage`, ContactEmail: `peter@cockroachlabs.com`,
+	OwnerStorage: {SlackRoom: `storage`, Mention: []string{`@cockroachdb/storage`},
```

Comment:
i've noticed bot mentions don't actually tag anyone 😢.

you can try though. I was going to submit a later PR that (a) finds members in that team and (b) cc them all.
i would also suggest merging this with TEAMS.yaml, and using the `internal/team` package to do this kind of thing. make sure it's all in one place. i realise that's more than what this PR intended though!

Reply:
> I was going to submit a later PR that (a) finds members in that team and (b) cc them all.

That's a good idea.

@cockroachdb/kv-east at least does something, though. You're saying it's not sending notifications?

> i would also suggest merging this with TEAMS.yaml, and using the internal/team package to do this kind of thing. make sure it's all in one place. i realise that's more than what this PR intended though!

It's what I'm intending to do! The main contribution here is cleaning up the issues code.

Reply:
When a bot does it, it doesn't work. Blathers example: https://github.com/cockroachdb/cockroach/issues/62799#issuecomment-810353480

Reply:
Unless it's a permissions problem....
I've tried asking for more perms from blathers.

Reply:
I'll ask the kv-east folks if they got a ping from the above.

Reply:
Theu did.

Reply:
ah okay, this is an issue that only affects blathers because it is an app. cockroach-teamcity is a user:

![image](https://user-images.githubusercontent.com/3646147/113619756-ba530680-969c-11eb-8dd8-52d8c70ea0ed.png)